doc/base/classes.xml

```diff
@@ -12365,6 +12365,18 @@
 				Return the list of keys in the [Dictionary].
 			</description>
 		</method>
+        <method name="parse_json">
+			<return type="int">
+			</return>
+			<argument index="0" name="json" type="String">
+			</argument>
+			<description>
+				Parse json string into the dictionary. Return OK when successed or the error code when failed.
+
+                Be aware that the JSON specification does not define integer or float types, but only a number type.
```

Comment:
Here too. Also, the empty newline above will be interpreted as an empty paragraph in the in-editor help and the online docs.